govcd/catalog.go

```diff
@@ -810,3 +810,40 @@ func (catalog *Catalog) QueryMediaList() ([]*types.MediaRecordType, error) {
 	}
 	return mediaResults, nil
 }
+
+// getOrgInfo finds the organization to which the catalog belongs, and returns its name and ID
+func (catalog *Catalog) getOrgInfo() (orgInfoType, error) {
```

Comment:
For me seems that `(adminCatalog *AdminCatalog) getOrgInfo()` is very similar. They only difference from where slise of `Links` are taken and which cache is used.

Reply:
code simplified